src/render.rs

```diff
@@ -11,7 +11,7 @@ pub struct MarkdownRenderer<'a> {
 
 impl<'a> MarkdownRenderer<'a> {
     /// Creates a new renderer instance.
-    pub fn new() -> MarkdownRenderer<'a> {
+    pub fn new(repo: Option<&'a str>) -> MarkdownRenderer<'a> {
```

Comment:
Should this parameter be renamed to something like `base_url` to better reflect its usage ?
Also, its usage should be documented.

Reply:
Will do!